_includes/page_footer.html

```diff
@@ -7,7 +7,7 @@
     <a href="/use-cases">Use Cases</a>&ensp;·&ensp;
     <a href="/druid-powered">Powered by Druid</a>&ensp;·&ensp;
     <a href="/docs/latest">Docs</a>&ensp;·&ensp;
-    <a href="/community/">Community</a>&ensp;·&ensp;
+    <a href="https://druid.apache.org/community">Community</a>&ensp;·&ensp;
```

Comment:
This should be https://druid.apache.org/community/ (/community is a redirect so it cannot be canonical).

Reply:
Fixed